prometheus/src/chart.tsx

```diff
@@ -2,29 +2,31 @@ import { EmptyContent, Loader } from '@kinvolk/headlamp-plugin/lib/CommonCompone
 import { Box } from '@material-ui/core';
 import { useEffect, useState } from 'react';
 import { Area, AreaChart, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
-import { fetchMetrics } from './request';
 
-export function Chart(props: {
+export interface ChartProps {
   plots: Array<{
     query: string;
     name: string;
     fillColor: string;
     strokeColor: string;
     dataProcessor: (data: any) => any[];
   }>;
+  fetchMetrics: (query: object) => Promise<any>;
```

Comment:
We should avoid having `any` types unless we really accept/expect anything.

Reply:
it is the type of request function return which is itself any

Reply:
updated

Reply:
That's... not really what we should expect but let's take it just so we can move on.